helm/ovn-kubernetes/values-single-node-zone.yaml

```diff
@@ -0,0 +1,162 @@
+# Values for ovn-kubernetes with single-node zone interconnect
+# Requires: ovnkube-single-node-zone, ovnkube-control-plane
+
+# -- list of dependent subcharts that need to be installed for the given deployment mode, these subcharts haven't been tested yet.
+tags:
+  ovn-ipsec: false
+  ovnkube-db: false
+  ovnkube-db-raft: false
+  ovnkube-master: false
+  ovnkube-node: false
+  ovnkube-node-dpu: false
+  ovnkube-node-dpu-host: false
+  ovnkube-zone-controller: false
+  ovnkube-single-node-zone: true
```

Comment:
nit: may not need line 14 here.

Reply:
somehow these lines are needed, otherwise the required components won't be created, this is not consistent with document, but I have no idea why it behaves like this yet.

Reply:
Helm, by design, merges the contents of values.yaml with any additional values files you specify using the -f flag. This means that even if you specify values-single-node-zone.yaml, Helm still considers the default values from values.yaml unless it is explicitly removed or left empty.

So we need to either temporarily remove or empty the contents of `values.yaml` to keep it from being read and making you have to set the tags back to true. Another idea is to rename values.yaml to something like `values-no-ic.yaml`

Reply:
good to know this, I rename `values.yaml` to `values-no-ic.yaml`.